### PR TITLE
Changelog for v0.5.1

### PR DESCRIPTION
**Pull request by RichVanderwal**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+## [0.5.1] - 2020-12-16
+- Fixed bug that resulted in payload size remaining slightly too large after
+splitting it into more manageable chunks.
```
Comment (MrAlias) [inline CHANGELOG.md:11]: Is there a PR/Issue number we can add to this so users interested in the details have a place for more info?

```suggestion
### Fixed
- Resolved bug that resulted in payload size remaining slightly too large after
splitting it into more manageable chunks.
```

Comment (RichVanderwal) [reply CHANGELOG.md]: Thanks for catching that! Fixed.

**Review by justinfoote — APPROVED** [review]
🚀

File: CHANGELOG.md (hunk 1 above)
Comment (MrAlias) [reply CHANGELOG.md]: Not a huge deal, but there were two parts to my comment:

1. correct the section header and adjust the sentence to account for it being in the `Fixed` section.
2. Add a link to the associated change.

I don't think 1 was addressed.

**Pull request merged**